### DISCUSSION TITLE
Separate no-NSFW and NSFW pictures

### DISCUSSION THREAD

**Discussion by TroyVi** [civitai/civitai#441] [Ideas]
Too often I search for models and half is blurred since they have nsfw pictures. I have no problems with nsfw pictures, but there's a time and place for everything. And it's not optimal to hide good models because one usage is to create nsfw pictures. My suggestion is to use the first non-nsfw picture as the thumbnail. And instead tag them as nsfw with text if it's appropriate. In the model gallery, I suggest that non-nsfw and nsfw pictures are separated in different tabs. A counter could be useful to see how many pictures there are in each category.

**Comment by Judas87Swiff** [comment]
Good idea, I second this, always having to unblur just to see a "perfectly" normal image of: Honoka (DOA), Mei (Overwatch), Tea Gardner (Anzu Mazaki), Lola Bunny, Sonico (Tsuji Santa), Rebecca Hopkins (Yu-Gi-Oh!), Orihime Inoue, Purah (Real Age) (Legend of Zelda), (Usually this is quite common with female characters...) as shown by this not extensive list.
Plus it just make it more clear at a quick glance what a model even is, some names are so random.
